src/pages/category/Index.vue

```diff
@@ -1,43 +1,86 @@
  <template>
   <div>
-    <a-row :gutter="24">
+    <a-row :gutter="24" style>
       <a-col :span="6">
-        <a-input-search style="margin-bottom: 8px" placeholder="Search" @change="onChange" />
-        <a-tree
-          :expanded-keys="expandedKeys"
-          :auto-expand-parent="autoExpandParent"
-          :show-line="true"
-          :tree-data="categories"
-          :replaceFields="{children:'children', title:'name', key:'id' }"
-          @expand="onExpand"
+        <a-card title="分类列表" style="height:calc(100vh - 164px)">
+          <a slot="extra" href="#">新增</a>
+          <a-tree
+            :tree-data="categories"
+            :replaceFields="{children:'children', title:'name', key:'id' }"
+            autoExpandParent
+          >
+          </a-tree>
+        </a-card>
+      </a-col>
+      <a-col :span="12" style="margin-top:50px;">
```

Comment:
is it possible to separate this part of UI into a single children component?

Reply:
If 'YES', also the data and method related/bind to this part should be wrapped  too.

Reply:
split as a single component will make code very diffcult. So it will be pending.